app/src/main/java/com/example/tapp/Record.java

```diff
@@ -27,19 +31,47 @@ private void handleTimerComplete() {
         switch (this.timersComplete) {
             case 1:
                 // Clear screen and set screen to waiting period.
-                timerLabel.setText("Please switch to your right hand.");
+                trial++;
+                timerLabel.setText("Please repeat " + hand + " hand for trial " + trial + ".");
                 countText.setText("");
                 this.waitingTimer.start();
+                lTotal += lCount;
+                lCount = 0;
+                rTotal += rCount;
+                rCount = 0;
                 break;
             case 2:
                 // End the waiting period and allow the next tap to start the timer.
-                timerLabel.setText("Tap the screen with your right hand.\nYou have 10 seconds to tap as quickly as possible.");
+                timerLabel.setText("Tap the screen with your " + hand + " hand.\nYou have 10 seconds" +
+                        " to tap as quickly as possible.\n" + hand + trial);
```

Comment:
Watch out for debugging statements like this in the future. (You should have caught this when testing after you finished writing the code.) @DesAndrew